easybuild/easyconfigs/b/binutils/binutils-2.26.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'binutils'
+version = '2.26'
+
+homepage = 'http://directory.fsf.org/project/binutils/'
+description = "binutils: GNU binary utilities"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+builddependencies = [
+    ('flex', '2.6.0'),
+    ('Bison', '3.0.4'),
+    # zlib required, but being linked instatically, so not a runtime dep
+    ('zlib', '1.2.8'),
+]
+
+# statically link with zlib, to avoid runtime dependency on zlib
+# further, add the system library path in the rpath: this should 'harden' the
+# resulting binutils to bootstrap GCC (no trouble when other libstdc++ is build etc)
+preconfigopts = 'LIBS="$EBROOTZLIB/lib/libz.a -Wl,-rpath=/lib64 -Wl,-rpath=/usr/lib64 -Wl,-rpath=/lib -Wl,-rpath=/usr/lib"'
```

Comment:
A sanity check for it would be nice. A simple `ldd bin/* | grep -c libz` would do.